lib/generators/sufia/work/templates/locale.en.yml.erb

```diff
@@ -1,4 +1,10 @@
 en:
+  sufia:
+    select_type:
+      <%= file_name %>:
+        name:               "<%= human_name.titleize %>"
+        description:        "<%= human_name %> works"
```

Comment:
This default seems a bit unuseful.  How about `A work to store a <%= human_name %>`  The you would get 'A work to store a Book' vs 'Book works' 'A work to store a Generic Work' vs 'Generic Work works'